profile/edit/index.html

```diff
@@ -0,0 +1,267 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Wall | Me & My Friends</title>
+    <script
+      src="https://kit.fontawesome.com/48de792fa6.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,400;0,600;0,700;0,800;1,400;1,600;1,700;1,800&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="/dist/css/styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <script src="/src/js/script.js" type="module"></script>
+    <script src="/node_modules/bootstrap/dist/js/bootstrap.bundle.min.js"></script>
+  </head>
+  <body style="--sidebar-width-open: 40vw">
```

Comment:
Why can't you do this in SCSS?

Reply:
on my website i am using a side-bar for a friendlist. i am using the same sidebar but replacing the friendlist with an edit profile form on "edit profile". the width is different and unique for this page. this way seemed like the least amount of effort required to get the result i wanted. i could maybe change the variable in js?